stable/ark-server/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
appVersion should be 0.7.1

Reply:
Indeed, and I would add that `version` should be at `1.0.0`, because it's the first version of this chart.

Reply:
We'll be releasing Ark v0.8.0 later this week. We'll be back to a single namespace instead of two. I'd recommend waiting until then before merging this chart if you can!